scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/spi/model/validator/CatalogObjectValidator.java

```diff
@@ -56,6 +70,49 @@ public String validate(String parameterValue, ModelValidatorContext context) thr
             throw new ValidationException("Expected value should match regular expression " + pattern.pattern() +
                                           " , received " + parameterValue);
         }
+        if (context == null || context.getSessionId() == null || context.getSessionId().isEmpty()) {
+            // Sometimes the workflow is parsed and checked without scheduler instance (e.g., submitted from catalog).
+            // In this case, we don't have the access of the scheduler global dataspace, so the validity check is passed.
+            logger.debug(String.format("Can't check the validity of the variable value [%s], because missing the access to the catalog",
```

Comment:
"Validity of the variable value [%s] is skipped because access to the catalog is disabled"